.github/workflows/test_filenames_check.yml

```diff
@@ -36,5 +36,6 @@ jobs:
           name-patterns: '*UnitTests.*,*IntegrationTests.*'
           paths: '**/src/test/scala/**'
           report-format: 'console'
+          excludes: 'server/src/test/scala/za/co/absa/atum/server/api/TestData.scala,server/src/test/scala/za/co/absa/atum/server/api/TestTransactorProvider.scala,server/src/test/scala/za/co/absa/atum/server/ConfigProviderTest.scala'
```

Comment:
is it possiblE to add multi-line exclusions instead of 1 line? Also, is it possible to regexp it? Or at least use wildcard(s)?

Reply:
For multi-line exclusion, see and use filename-inspector branch `feature/fix-to-use-correct-syntax` in your yaml.
- See updated README.md for an example.

Regex
- paths support glob patterns (supports **)
- name-patterns and excludes support fnmatch pattern (do not support **)

Reply:
I think wildcards are possible. RegExpt no, at least not yet now (it hasn't been released yet even).
I actually thought to move the classes into own directories, to make the exclusions simpler. But in its own PR.

Reply:
Add wildcards, here is my testing example:

```yaml
- name: Test filename-inspector check
  id: test-file-suffix
  uses: AbsaOSS/filename-inspector@feature/fix-to-use-correct-syntax
  with:
    name-patterns: |
      *GUI*,
      *Unit*
    paths: |
      **/src/test/java/**,
      **/src/test/scala/**
    excludes: |
      *1UnitTest.*,
      *thServiceTest.*
    report-format: 'csv'
    verbose-logging: 'true'
    fail-on-violation: 'true'
```

Reply:
The multiline example doesn't seem to work. 😠 Reverting to single line, that worked.
(Tried all kind of formats)